Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2999,6 +3000,15 @@ extension BrowserViewController: IntroViewControllerDelegate {
             }
         }
     }
+    
+    func introViewControllerShouldAskForNotifications(_ introViewController: IntroViewController) -> Bool {
+        // Only include the slide if Leanplum has decided it needs to run and if the current locale is en_US/GB/CA/AU/HK/SG
+        return Leanplum.hasStarted() && Locale.current.identifier.startsWith("en_")
+    }
+    
+    func introViewControllerDidRequestToEnableNotifications(_ introViewController: IntroViewController) {
+        // TODO Calling into the FxaLoginHelper shows the permissions dialog but then crashes
+    }
```

Comment:
`FxALoginHelper` assumes that the `profile` has been set and that the user has logged into the their `FirefoxAccount`.

Second, the APNS token we get back from Apple needs to be given to Leanplum's servers.

Currently, we're not storing this anywhere; I'm not sure if we can re-use the same token for both FxA and Leanplum.

Probably the best thing for us to do here is set some `FxALoginHelperDelegate` methods.